<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DcDialog</class>
 <widget class="QDialog" name="DcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>直流</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/Icon/Icon/wave-1.png</normaloff>:/Icon/Icon/wave-1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="DcWaveOhm_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>阻抗选择:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="buddy">
         <cstring>DcWaveNo_Cmb</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DcOhm_Cmb">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Andalus</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="DcWaveNo_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>波形类型:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="buddy">
         <cstring>DcWaveNo_Cmb</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DcWaveNo_Cmb">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Andalus</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="StandBy_pushBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}

</string>
   </property>
   <property name="text">
    <string>STANDBY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutputOn_pushBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
 
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>OUTPUTON</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(0, 170, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="DcAmp_Label">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>幅值:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="DcAmp_Dsb">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>149.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DcAmpUnit_Cmb">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Batang</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>true</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Outline</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="DcAmp_Dial">
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="notchesVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
